src/unity/toolkits/evaluation/metrics.cpp

```diff
@@ -227,7 +296,7 @@ variant_map_type compute_classifier_metrics_from_probability_vectors(
     }
     return false;  // Never stop before all rows have been tallied.
   };
-  data.materialize_to_callback(callback);
+  data.materialize_to_callback(callback, turi::thread::cpu_count());
```

Comment:
This resolves a scary bug that somehow hadn't cropped up before: the evaluators default to allocating buffers to support `turi::thread::cpu_count` parallelism (== 8 on my MBP), but `gl_sframe::materialize_to_callback` defaults to `SFRAME_DEFAULT_NUM_SEGMENTS`, which for some reason was 16 during testing.